run_dir/static/js/pricing_main.js

```diff
@@ -379,10 +379,10 @@ const vPricingMain = {
             axios
                 .get(url)
                 .then(response => {
-                    this.USD_in_SEK = response.data.USD_in_SEK
-                    this.EUR_in_SEK = response.data.EUR_in_SEK
+                    this.USD_in_SEK = response.data.USD_in_SEK.toFixed(2)
+                    this.EUR_in_SEK = response.data.EUR_in_SEK.toFixed(2)
```

Comment:
I don't think we want this right? Keeping the decimals is a good thing.

Reply:
Done!